src/budy/models/account.py

```diff
@@ -182,7 +182,10 @@ def set_store_s(self, store):
     @appier.operation(name = "Notify")
     def notify(self, name = None):
         name = name or "account.new"
-        account = self.reload(map = True)
+        account = self.reload(

```

Comment:
rules is True to be able to access the confirmation token, otherwise I won't be able to create the confirmation url